server/controllers/stock/reports/stock_inline_movements.report.handlebars

```diff
@@ -0,0 +1,85 @@
+{{> head title="TREE.STOCK_MOVEMENTS" }}
+
+<body>
+
+  {{> header}}
+
+  <!-- body  -->
+  <div class="row">
+    <div class="col-xs-12">
+
+      <!-- page title  -->
+      <h2 class="text-center text-uppercase">
+        {{translate 'TREE.STOCK_INLINE_MOVEMENTS'}}
+      </h2>
+
+      <h4 class="text-center">
+        {{date}}
+      </h4>
+
+      <!-- filters  -->
+      {{> filterbar filters=filters }}
+
+      <!-- list of data  -->
+      <table class="table table-condensed table-bordered table-report">
+        <thead>
+          <tr>
+            <th>{{translate 'TABLE.COLUMNS.REFERENCE'}}</th>
+            <th>{{translate 'STOCK.IO'}}</th>
+            <th>{{translate 'STOCK.COST'}}</th>
+            <th>{{translate 'FORM.LABELS.DATE'}}</th>
+            <th>{{translate 'STOCK.FLUX'}}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {{!-- for each depots which contains inventories --}}
+          {{#each depots as | items name |}}
+
+            <!-- this is the depot group header -->
+            <tr style="border:none">
+              <th style="border:none; border-bottom: solid black 2px;" class="text-uppercase">
+                {{ name }}
+              </th>
+              <th colspan="4" style="border:none; border-bottom: solid black 2px;" class="text-right">
+                ({{ items.length }} {{ translate "TABLE.AGGREGATES.RECORDS" }})
+              </th>
+            </tr>
+
+            <!-- these are the items for each group -->
+            {{#each items}}
+              <tr {{#if is_exit}}class="text-danger"{{/if}}>
+                <td>{{documentReference}}</td>
+                <td>
+                  {{#if is_exit}}
+                    {{translate 'STOCK.OUTPUT'}}
+                  {{else}}
+                    {{translate 'STOCK.INPUT'}}
+                  {{/if}}
+                </td>
+                <td class="text-right">{{currency cost ../../metadata.enterprise.currency_id}}</td>
+                <td>{{date date}}</td>
+                <td>{{translate flux_label}}</td>
+              </tr>
+            {{/each}}
+
+            <!-- blank row  -->
+            {{#unless @last }}
+              <!-- blank line -->
+              <tr style="border:none;">
+                <th style="border:none;"></th>
+              </tr>
+            {{/unless}}
+
+          {{else}}
+            {{> emptyTable columns=6}}
+          {{/each}}
+        </tbody>
+        <tfoot>
+          <tr style="font-weight: bold;">
+            <td colspan="5">{{rows.length}} {{translate 'STOCK.ITEMS'}}</td>
```

Comment:
![image](https://user-images.githubusercontent.com/5445251/84618927-8ddebb80-aecb-11ea-9608-6609072fb9c7.png)

In case of multiple depots, value in the line of the depot name is the total of the depot and the value in the bottom is the total of items loaded for all depots

Reply:
I agree.